2025-05-04 10:30:30,997:DEBUG:certbot._internal.main:certbot version: 2.10.0
2025-05-04 10:30:30,997:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/bin/certbot
2025-05-04 10:30:30,997:DEBUG:certbot._internal.main:Arguments: ['-v', '--webroot', '--agree-tos', '--renew-by-default', '--preferred-challenges', 'http-01', '--email', 'vaxelaire.yohem@gmail.com', '--webroot-path', '/var/www/certbot', '-d', 'hackthebox.vaxelaire.fr', '-n']
2025-05-04 10:30:30,997:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2025-05-04 10:30:31,051:DEBUG:certbot._internal.log:Root logging level set at 20
2025-05-04 10:30:31,051:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2025-05-04 10:30:31,052:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A seperate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7fcbeba87a10>
Prep: True
2025-05-04 10:30:31,052:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7fcbeba87a10> and installer None
2025-05-04 10:30:31,052:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2025-05-04 10:30:31,100:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/2243553505', new_authzr_uri=None, terms_of_service=None), b674a225a17192af95a621d2ec1bc036, Meta(creation_dt=datetime.datetime(2025, 2, 22, 10, 55, 38, tzinfo=<UTC>), creation_host='faa0276bdd8e', register_to_eff=None))>
2025-05-04 10:30:31,100:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2025-05-04 10:30:31,102:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2025-05-04 10:30:31,504:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 1012
2025-05-04 10:30:31,504:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 04 May 2025 10:30:31 GMT
Content-Type: application/json
Content-Length: 1012
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "LY6Xlunke3A": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "profiles": {
      "classic": "https://letsencrypt.org/docs/profiles#classic",
      "shortlived": "https://letsencrypt.org/docs/profiles#shortlived (not yet generally available)",
      "tlsserver": "https://letsencrypt.org/docs/profiles#tlsserver"
    },
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.5-February-24-2025.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert"
}
2025-05-04 10:30:31,525:DEBUG:certbot._internal.renewal:Auto-renewal forced with --force-renewal...
2025-05-04 10:30:31,525:DEBUG:certbot._internal.display.obj:Notifying user: Renewing an existing certificate for hackthebox.vaxelaire.fr
2025-05-04 10:30:31,527:DEBUG:acme.client:Requesting fresh nonce
2025-05-04 10:30:31,527:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2025-05-04 10:30:31,659:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2025-05-04 10:30:31,660:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 04 May 2025 10:30:31 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: aJoEAs1IXp_eK1tEzAbjeTGl9eY6T824d1ZsbbgkBHAJHyZe7Jo
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2025-05-04 10:30:31,660:DEBUG:acme.client:Storing nonce: aJoEAs1IXp_eK1tEzAbjeTGl9eY6T824d1ZsbbgkBHAJHyZe7Jo
2025-05-04 10:30:31,661:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "hackthebox.vaxelaire.fr"\n    }\n  ]\n}'
2025-05-04 10:30:31,663:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjI0MzU1MzUwNSIsICJub25jZSI6ICJhSm9FQXMxSVhwX2VLMXRFekFiamVUR2w5ZVk2VDgyNGQxWnNiYmdrQkhBSkh5WmU3Sm8iLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "bLBSi6WU2Aue8TNUhuBg5SgPFgJK6Wg6E8AYiTuBh4MX82ZJOPLEIQ31Wnb65KhFfCPn42nWRe7ZGDmFJ8qQjIQCNy3qd4ygnfqVbb48qKQmwklB5wzeOWVuyHGK8_zoy0XE7lP-koeH7OTIOUcZx_9Ncfe8mugN8c-fFaN31S-K_Y-VcuGAW3NLCc3l1u6gCJ3naq4R3nGpx0-8a2125fdBtEE2zUmELnqEyXxppCKGDATqqihpmqKL0YhGGMmFbFgoMN2OEevD1l_zUhln_Mpnnx2jI8Y9QMiWRTlZokld1iW_AvjeFZ6qJ2EAb-4mJpgmZ5oySfNLDfQMbD_XWQ",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogImhhY2t0aGVib3gudmF4ZWxhaXJlLmZyIgogICAgfQogIF0KfQ"
}
2025-05-04 10:30:31,844:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 357
2025-05-04 10:30:31,844:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Sun, 04 May 2025 10:30:31 GMT
Content-Type: application/json
Content-Length: 357
Connection: keep-alive
Boulder-Requester: 2243553505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/2243553505/380699899947
Replay-Nonce: aJoEAs1I2H0d0ZY3N4aPQxfU2eFeHnsmYLaMAC2szAYzFxvaJ3c
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2025-05-11T10:30:31Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "hackthebox.vaxelaire.fr"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz/2243553505/515319305087"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/2243553505/380699899947"
}
2025-05-04 10:30:31,845:DEBUG:acme.client:Storing nonce: aJoEAs1I2H0d0ZY3N4aPQxfU2eFeHnsmYLaMAC2szAYzFxvaJ3c
2025-05-04 10:30:31,845:DEBUG:acme.client:JWS payload:
b''
2025-05-04 10:30:31,846:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2243553505/515319305087:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjI0MzU1MzUwNSIsICJub25jZSI6ICJhSm9FQXMxSTJIMGQwWlkzTjRhUFF4ZlUyZUZlSG5zbVlMYU1BQzJzekFZekZ4dmFKM2MiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIyNDM1NTM1MDUvNTE1MzE5MzA1MDg3In0",
  "signature": "Ccc0bx3M38q54ArTBtWknf4z4zWayHi4o2WrChheFvmKnw6D_k93sslRQTpk2tdwawKJ62Zn0kmy2CtnPymEjPK9cJ6osm0iB6cne5WGVjh_ZBiH8E596HqI6mwVv9O3rIDoRB86yhlJ83bMHunQW-JSKVYZAvL-47_0dSPdZtTiFbozKHSlaVahfy0Scka87BshOCE13a4QSzTZOG2EAdNgnldnLzb2e0wE29UCr4I6Ex5wB1HS3yJC74LP569WvBpq4cRJBNmmwL3MYkojoMTR-RZ7uFgWMtbhR_OQ_Lcv2zJn019Cm5s3Vb8gvO7H0-NI7SNM7MveSt_bc6OGKw",
  "payload": ""
}
2025-05-04 10:30:31,989:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2243553505/515319305087 HTTP/1.1" 200 831
2025-05-04 10:30:31,989:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 04 May 2025 10:30:31 GMT
Content-Type: application/json
Content-Length: 831
Connection: keep-alive
Boulder-Requester: 2243553505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: FIfrddYQJ9VotAOFqPnZqHRYLo5rWFbLuFzkOdqsYtlVzbk1Nng
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "hackthebox.vaxelaire.fr"
  },
  "status": "pending",
  "expires": "2025-05-11T10:30:31Z",
  "challenges": [
    {
      "type": "dns-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/515319305087/nBTtSw",
      "status": "pending",
      "token": "mxmcPngOldDrnyBCHEHt-AUd1sT33cCxT2_O0xWIdfI"
    },
    {
      "type": "tls-alpn-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/515319305087/6O3lFw",
      "status": "pending",
      "token": "mxmcPngOldDrnyBCHEHt-AUd1sT33cCxT2_O0xWIdfI"
    },
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/515319305087/iGz74Q",
      "status": "pending",
      "token": "mxmcPngOldDrnyBCHEHt-AUd1sT33cCxT2_O0xWIdfI"
    }
  ]
}
2025-05-04 10:30:31,989:DEBUG:acme.client:Storing nonce: FIfrddYQJ9VotAOFqPnZqHRYLo5rWFbLuFzkOdqsYtlVzbk1Nng
2025-05-04 10:30:31,990:INFO:certbot._internal.auth_handler:Performing the following challenges:
2025-05-04 10:30:31,990:INFO:certbot._internal.auth_handler:http-01 challenge for hackthebox.vaxelaire.fr
2025-05-04 10:30:31,990:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/certbot for all unmatched domains.
2025-05-04 10:30:31,990:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/certbot/.well-known/acme-challenge
2025-05-04 10:30:31,991:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/certbot/.well-known/acme-challenge/mxmcPngOldDrnyBCHEHt-AUd1sT33cCxT2_O0xWIdfI
2025-05-04 10:30:31,992:DEBUG:acme.client:JWS payload:
b'{}'
2025-05-04 10:30:31,993:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/515319305087/iGz74Q:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjI0MzU1MzUwNSIsICJub25jZSI6ICJGSWZyZGRZUUo5Vm90QU9GcVBuWnFIUllMbzVyV0ZiTHVGemtPZHFzWXRsVnpiazFObmciLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLzIyNDM1NTM1MDUvNTE1MzE5MzA1MDg3L2lHejc0USJ9",
  "signature": "WiuXL1PQPK_pHSdL0iEmHowVY4W2ycakxITYJKfokxX2-u-7q7FhA59o0Yy3I7bcWKy0O6vt2EayNmFovCPgpzElki9cGe5j3RsUtUkTX4jq-r_FxYTUmJh7BZNQvcYTiwmJnx3mIQGbrpSHKroDvjtBN5ljKqO0SvQnyEs0Kl0u5GXlIWroorkX5o_ot3vUUXCitPxcn0RNxmDApSFWbLHUQPvcataPZP0z6-MLBhcgmG_D8eXFgcFpNkN3vCVAg459eUgtfSbKx2Tl0jKrlKU400BaxqLDEY2NY102GtsTGuAi0tCja7I8yowTlq7MFHrCcMQ4-DjM4RzEjlG0Lg",
  "payload": "e30"
}
2025-05-04 10:30:32,131:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall/2243553505/515319305087/iGz74Q HTTP/1.1" 200 195
2025-05-04 10:30:32,132:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 04 May 2025 10:30:32 GMT
Content-Type: application/json
Content-Length: 195
Connection: keep-alive
Boulder-Requester: 2243553505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz/2243553505/515319305087>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/515319305087/iGz74Q
Replay-Nonce: aJoEAs1IlCNKAb7MRTVMJbqM7H0JJydEKTlLFq7gLeY93_iVg30
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/515319305087/iGz74Q",
  "status": "pending",
  "token": "mxmcPngOldDrnyBCHEHt-AUd1sT33cCxT2_O0xWIdfI"
}
2025-05-04 10:30:32,132:DEBUG:acme.client:Storing nonce: aJoEAs1IlCNKAb7MRTVMJbqM7H0JJydEKTlLFq7gLeY93_iVg30
2025-05-04 10:30:32,132:INFO:certbot._internal.auth_handler:Waiting for verification...
2025-05-04 10:30:33,132:DEBUG:acme.client:JWS payload:
b''
2025-05-04 10:30:33,134:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2243553505/515319305087:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjI0MzU1MzUwNSIsICJub25jZSI6ICJhSm9FQXMxSWxDTktBYjdNUlRWTUpicU03SDBKSnlkRUtUbExGcTdnTGVZOTNfaVZnMzAiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIyNDM1NTM1MDUvNTE1MzE5MzA1MDg3In0",
  "signature": "mXxhgfQ3NiYRV9m5dwidCnfPtwDmIw3H5LwcuNfLRaWbTka68FOJ6mqY2wqX3oMF5UvQiW3Bth_T6BvL7uHhKB3lT_T9po9OCpI_D0zy781COVQOZZ-CxdFfWf5Vok7F5j0pZQpQpVa5RVaS_f2Kn8L-vWi9f5GQ6koXty0LUlLoqN0MJA16bjXomfKdzzdBh_CYPgpcz-NwstVqB4OJdnkiZQTc_YCQ7FUqEq10OsRqwo4qyoljFu7RrBoTgSkwUl2K0qoGIgarJL6m5xs3hJLpSjx6h_hbPBXxq7cAVvE764NSgagdJAgP_XEJPe6q0y8NqJ5M0HBWQ0KWMwCoVw",
  "payload": ""
}
2025-05-04 10:30:33,270:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2243553505/515319305087 HTTP/1.1" 200 1489
2025-05-04 10:30:33,270:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 04 May 2025 10:30:33 GMT
Content-Type: application/json
Content-Length: 1489
Connection: keep-alive
Boulder-Requester: 2243553505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: aJoEAs1IVpG4cGRbcZYRvrCuBWDwc0mrPi7GMZ2vBJVi_ZUzwic
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "hackthebox.vaxelaire.fr"
  },
  "status": "invalid",
  "expires": "2025-05-11T10:30:31Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/515319305087/iGz74Q",
      "status": "invalid",
      "validated": "2025-05-04T10:30:32Z",
      "error": {
        "type": "urn:ietf:params:acme:error:connection",
        "detail": "91.108.113.88: Fetching http://hackthebox.vaxelaire.fr/.well-known/acme-challenge/mxmcPngOldDrnyBCHEHt-AUd1sT33cCxT2_O0xWIdfI: Error getting validation data",
        "status": 400
      },
      "token": "mxmcPngOldDrnyBCHEHt-AUd1sT33cCxT2_O0xWIdfI",
      "validationRecord": [
        {
          "url": "http://hackthebox.vaxelaire.fr/.well-known/acme-challenge/mxmcPngOldDrnyBCHEHt-AUd1sT33cCxT2_O0xWIdfI",
          "hostname": "hackthebox.vaxelaire.fr",
          "port": "80",
          "addressesResolved": [
            "91.108.113.88",
            "2a02:4780:28:d15a::1"
          ],
          "addressUsed": "2a02:4780:28:d15a::1"
        },
        {
          "url": "http://hackthebox.vaxelaire.fr/.well-known/acme-challenge/mxmcPngOldDrnyBCHEHt-AUd1sT33cCxT2_O0xWIdfI",
          "hostname": "hackthebox.vaxelaire.fr",
          "port": "80",
          "addressesResolved": [
            "91.108.113.88",
            "2a02:4780:28:d15a::1"
          ],
          "addressUsed": "91.108.113.88"
        }
      ]
    }
  ]
}
2025-05-04 10:30:33,270:DEBUG:acme.client:Storing nonce: aJoEAs1IVpG4cGRbcZYRvrCuBWDwc0mrPi7GMZ2vBJVi_ZUzwic
2025-05-04 10:30:33,271:INFO:certbot._internal.auth_handler:Challenge failed for domain hackthebox.vaxelaire.fr
2025-05-04 10:30:33,271:INFO:certbot._internal.auth_handler:http-01 challenge for hackthebox.vaxelaire.fr
2025-05-04 10:30:33,271:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: hackthebox.vaxelaire.fr
  Type:   connection
  Detail: 91.108.113.88: Fetching http://hackthebox.vaxelaire.fr/.well-known/acme-challenge/mxmcPngOldDrnyBCHEHt-AUd1sT33cCxT2_O0xWIdfI: Error getting validation data

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2025-05-04 10:30:33,272:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/usr/lib/python3.12/site-packages/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2025-05-04 10:30:33,272:DEBUG:certbot._internal.error_handler:Calling registered functions
2025-05-04 10:30:33,272:INFO:certbot._internal.auth_handler:Cleaning up challenges
2025-05-04 10:30:33,272:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/certbot/.well-known/acme-challenge/mxmcPngOldDrnyBCHEHt-AUd1sT33cCxT2_O0xWIdfI
2025-05-04 10:30:33,272:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2025-05-04 10:30:33,272:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/bin/certbot", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/main.py", line 1894, in main
    return config.func(config, plugins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/main.py", line 1600, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/main.py", line 131, in _get_and_save_cert
    renewal.renew_cert(config, domains, le_client, lineage)
  File "/usr/lib/python3.12/site-packages/certbot/_internal/renewal.py", line 399, in renew_cert
    new_cert, new_chain, new_key, _ = le_client.obtain_certificate(domains, new_key)
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/usr/lib/python3.12/site-packages/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2025-05-04 10:30:33,275:ERROR:certbot._internal.log:Some challenges have failed.
